multigeiger/multigeiger.ino

```diff
@@ -532,6 +537,9 @@ void loop()
 
     Dose_Rate = Count_Rate *GMC_factor_uSvph;                        // ... and dose rate
 
+    // indicate status on RGB LED (if any)
+    indicate(Dose_Rate, I_TEST);
```

Comment:
just put it there for now.

in the end, i guess it should end up in a block that gets triggered every second.

Reply:
The TEST indication is also just for now for testing and should be replaced by a indication computed from other geiger status values, e.g. whether HV generation has problems, WLAN/LoRa has problems, etc.